blog/2020-12-21-openhab-3-0-release.md

```diff
@@ -0,0 +1,89 @@
+
+## Architecture Evolution
+
+Version 1 of openHAB was in itself a clean and simple architecture that worked well up to the moment that more and more "regular" users joined the community and complemented the rather dev-centric crowd.
+
+Version 2 of openHAB had to do compromises between the "old" purely textual openHAB 1 world and the new UI-enabling features that were introduced with Eclipse SmartHome. The way one had to work with bindings and rules very much differed from each other and these two world existed more or less side-by-side, glued together through the 1.x compatibility layer.
+
+A huge step of simplification in openHAB 3 is therefore the removal of the 1.x compatibility layer. All bindings now work exactly the same way and the user decides on the preference of configuration done textually or through the UI. To make sure that nobody is left behind by this move, [many 1.x bindings have been reimplemented](https://github.com/openhab/openhab-addons/issues/6179#migrated) for openHAB 3. For the rest, it is possible to keep an old 2.x instance running while integrating it into a new 3.0 setup - see the [openHAB outposts](#outposts) below.
+
+Furthermore, the two separate rule engine implementations have been merged into one, so that there is now a single concept for automation rules. As a consequence, textual rules written in openHAB's own Rule DSL also turn up within the user interface. 
+
+Besides the visible changes, many refactorings have been done to the codebase in order to keep it modern and maintainable. Moving from Java 8 to Java 11 brings the language features on a current level and ensures that openHAB continues to run on JVMs that receive security patches. Additionally, many used libraries have been upgraded, deprecated methods were removed, null annotations introduced, newer OSGi features used and so on. Overall, the codebase is future-proof and a solid foundation to further build on for the years to come.
+
+<p align="center"><video autoplay loop src="/uploads/evolution.mp4"></video></p>
+
+### Outposts
+
+With the new [openHAB Remote Binding](https://www.openhab.org/addons/bindings/remoteopenhab/) it is now very easy to create distributed setups. While in the past many users set up an MQTT broker and integrated multiple openHAB instances using the MQTT binding, openHAB 3 is now capable of discovering other instances in the local network and automatically read its items and things through the REST API.
+
+Such a setup then consists out of a central openHAB instance and several openHAB outposts. This makes sense, if certain devices need physical vicinity, e.g. because they are connected through a serial interface or because their wireless range is not sufficient to reach the central instance like it can be the case for Bluetooth sensors.
```

Comment:
```suggestion
Such a setup then consists out of a central openHAB instance and several openHAB outposts. This makes sense, if certain devices need physical vicinity, e.g. because they are connected through a serial interface or because their wireless range is not sufficient to reach the central instance, as can be the case for Bluetooth and other short range sensors.
```